### PR TITLE
Make webhooks pool configurable

### PR DESCRIPTION
**Pull request by codeadict**
This is a fix for #612 . @larshesel  @dergraf  please review 🎉

### CODE REVIEW

File: apps/vmq_webhooks/priv/vmq_webhooks.schema [hunk 1]
```diff
@@ -83,3 +83,14 @@
  [{datatype, string},
   hidden,
   {default, "    webhooks    Manage webhooks\n"}]}.
+
+%% @doc Configue the maximum number of HTTP connections maintained in the webhooks pool.
```
Comment (codeadict) [inline apps/vmq_webhooks/priv/vmq_webhooks.schema]: Should these options be hidden?

File: THANKS [hunk 2]
```diff
@@ -4,7 +4,7 @@ THANKS A LOT
 ------------
 
 Johanna Appel
-codeadict
+Dairon Medina Caro (@codeadict)
```
Comment (larshesel) [inline THANKS]: I love this line!

File: apps/vmq_webhooks/priv/vmq_webhooks.schema [hunk 3]
```diff
@@ -83,3 +83,14 @@
...
+{mapping, "vmq_webhooks.hackney_pool_max_connections", "vmq_webhooks.hackney_pool_max_connections",
+ [{datatype, integer},
+  {default, 100}]}.
+
+%% @doc Configue the timeout that we keep the connection alive in the 
+%% webhooks connection pool, expressed in milliseconds. 
+{mapping, "vmq_webhooks.hackney_pool_timeout", "vmq_webhooks.hackney_pool_timeout",
```
Comment (larshesel) [inline apps/vmq_webhooks/priv/vmq_webhooks.schema]: I don't like exposing into the config that we're using hackney so I'd remove that prefix.

I think it a name like `idle_connection_timeout` or something like that would be better (timeout is too generic).

File: apps/vmq_webhooks/priv/vmq_webhooks.schema (hunk 1 above)
Comment (larshesel) [reply apps/vmq_webhooks/priv/vmq_webhooks.schema]: Configue  -> Configure

Yeah, I'd make them hidden so the vernemq.conf file doesn't get too large, but I'll add the options to the documentation.

File: apps/vmq_webhooks/priv/vmq_webhooks.schema [hunk 4]
```diff
@@ -83,3 +83,14 @@
  [{datatype, string},
   hidden,
   {default, "    webhooks    Manage webhooks\n"}]}.
+
+%% @doc Configue the maximum number of HTTP connections maintained in the webhooks pool.
+{mapping, "vmq_webhooks.hackney_pool_max_connections", "vmq_webhooks.hackney_pool_max_connections",
+ [{datatype, integer},
+  {default, 100}]}.
+
+%% @doc Configue the timeout that we keep the connection alive in the
```
Comment (larshesel) [inline apps/vmq_webhooks/priv/vmq_webhooks.schema]: Configue -> Configure

File: apps/vmq_webhooks/priv/vmq_webhooks.schema (hunk 3 above)
Comment (codeadict) [reply apps/vmq_webhooks/priv/vmq_webhooks.schema]: Great!  I will do that, perhaps `pool_connection_timeout`

File: apps/vmq_webhooks/priv/vmq_webhooks.schema (hunk 4 above)
Comment (codeadict) [reply apps/vmq_webhooks/priv/vmq_webhooks.schema]: oops

File: apps/vmq_webhooks/priv/vmq_webhooks.schema (hunk 3 above)
Comment (larshesel) [reply apps/vmq_webhooks/priv/vmq_webhooks.schema]: sounds good.

**Comment by codeadict** [comment]
Cool, squashed now!

**Comment by codeadict** [comment]
Weird , it failed in Erlang 19 :(
, seems not related to my code

**Comment by larshesel** [comment]
Yeah, that test is a bit flaky - have to spend some time making it stable soon. I restarted the build and will merge when it's green.

**Comment by larshesel** [comment]
Merged. I just realized I should have merged this via the command line (using --ff) and then pushed from there - the github merge didn't preserve your verified committer information :/ Sorry about that.

**Pull request merged**